app/jobs/scheduled/sync_salesforce_users.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module ::Jobs
+  class SyncSalesforceUsers < ::Jobs::Scheduled
+    every 12.hours
+    sidekiq_options retry: false
+
+    def execute(args)
+      return unless SiteSetting.salesforce_enabled
+
+      lead_fields = UserCustomField.where(name: ::Salesforce::Person::LEAD_ID_FIELD)
+      lead_fields.find_in_batches(batch_size: 100) do |fields|
+        ids = fields.pluck(:value)
+        data = Salesforce::Api.new.get("composite/sobjects/Lead?fields=ConvertedContactId&ids=#{ids.join(",")}")
```

Comment:
Maybe we should add a try...catch around this and the other API GET call, we might be able to complete other items in the batch then even if one of them fails